CONTRIBUTING.rst

```diff
@@ -10,7 +10,7 @@ design enhancements issues as well as pull Requests to make *coala* more
 convenient, functional and bug free.
 
```

Comment:
hey your commit messages could use some work. heres a guide to help you: coala.io/commit

in a nutshell, it should be like: ``Contributing.rst: Rename "coala-analyzer"->"coala"`` or so. please do the same to the other commits as well.

additionally, to automatically close issues we use "Closes issue_link", so instead of ref use Closes, and also please give the full link, not just the relative one as you just did :) Thanks!